### DISCUSSION TITLE
ServiceNow Develoepr - SECRET clearance

### DISCUSSION THREAD

**Discussion by itrusthcm** [community/community#142878] [Codespaces]
### Select Topic Area

Show & Tell

### Body

Hi, Please see 3 ServiceNow Developer opportunities in MD, UT, IL Air Force Base: 

https://itrusthcm.applytojob.com/apply/3AIyy1QIIo/Service-Now-Developer

https://itrusthcm.applytojob.com/apply/6XVw0utFxj/Service-Now-Developer

https://itrusthcm.applytojob.com/apply/2WOQHNT9QR/Service-Now-Developer

**Comment by ebndev** [comment]
Hi @itrusthcm, thanks for sharing with the community! 

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
